<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>superviser_mainioeratorfor</class>
 <widget class="QWidget" name="superviser_mainioeratorfor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Book.png</normaloff>:/Book.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;您现在的权限为 &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;管理员&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_explore"/>
         </item>
         <item>
          <widget class="QPushButton" name="bt_explore">
           <property name="text">
            <string>搜索</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/find.png</normaloff>:/find.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tw_library"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;外观设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ckb_rowhead">
           <property name="text">
            <string>行表头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ckb_columnhead">
           <property name="text">
            <string>列表头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ckb_color">
           <property name="text">
            <string>交替背景色</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_bookname">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;书名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_bookname"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_author">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;作者&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_author"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="bt_addbook">
         <property name="text">
          <string>添加书籍</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/add.png</normaloff>:/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_d">
         <property name="text">
          <string>删除书籍</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/canso.png</normaloff>:/canso.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_return">
         <property name="text">
          <string>退出</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/quit.png</normaloff>:/quit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
